Projects/UOContent/Multis/Boats/BaseBoat.cs

```diff
@@ -99,6 +99,14 @@ public BaseBoat(Serial serial) : base(serial)
         {
         }
 
+        [CommandProperty(AccessLevel.GameMaster)]
+        public bool IgnoreWater { get; set; }
+
+        [CommandProperty(AccessLevel.GameMaster)]
+        public bool Flies { get; set; }
+
+        public bool Flying { get; set; }
```

Comment:
Let's removing flying for now until we are ready to introduce it. Looks like the code is commented out.